### PR TITLE
platform-existing: update docs for deploying on existing k8s cluster

### PR DESCRIPTION
**Pull request by yanniszark**
Update docs for installing Kubeflow on an existing Kubernetes Cluster.

Signed-off-by: Yannis Zarkadas <yanniszark@arrikto.com>

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/833)

### CODE REVIEW

File: content/docs/started/getting-started-k8s.md [hunk 1]
```diff
@@ -4,80 +4,123 @@ description = "Instructions for installing Kubeflow on your existing Kubernetes
...
 
 ## Delete Kubeflow
 
 Run the following commands to delete your deployment and reclaim all resources:
 
 ```
 cd ${KFAPP}
-# If you want to delete all the resources, including storage.
-kfctl delete all --delete_storage
-# If you want to preserve storage, which contains metadata and information
```
[the remark this answers is not in the document]
Comment (yanniszark) [reply content/docs/started/getting-started-k8s.md]: Currently, only the GCP platform handles the particular flag.
It means that platform GCP will do additional API calls to delete the underlying Persistent Disks.
In the case of an existing cluster, this doesn't have much meaning.
Whether the storage is deleted or preserved will depend on the ReclaimPolicy of the storage class that the cluster uses.

File: content/docs/started/getting-started-k8s.md [hunk 2]
```diff
@@ -4,80 +4,123 @@ description = "Instructions for installing Kubeflow on your existing Kubernetes
...
+GCP_ZONE="us-west1-b"
+
+CLUSTER_VERSION="$(gcloud container get-server-config --format="value(validMasterVersions[0])")" 
+CLUSTER_NAME="kubeflow"
+
+############################
+# Create and setup cluster #
+############################
+
+gcloud beta container clusters create ${CLUSTER_NAME} \
```
[the remark this answers is not in the document]
Comment (yanniszark) [reply content/docs/started/getting-started-k8s.md]: Probably a remnant from old times :sweat_smile: 
This script has served me for a long time.
I will change it to `gcloud`.

**Comment by yanniszark** [comment]
@sarahmaddox thanks a lot for all the useful feedback!
I have incorporated the changes in a new commit, so that it's easier to check them.

File: content/docs/started/getting-started-k8s.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sarahmaddox) [reply content/docs/started/getting-started-k8s.md]: Thanks!

**Comment by sarahmaddox** [comment]
/lgtm
/approve

Thank you!

**Pull request merged**